actions/websocket_actions.jsx

```diff
@@ -479,7 +482,13 @@ export function handleEvent(msg) {
     case SocketEvents.CLOUD_PAYMENT_STATUS_UPDATED:
         dispatch(handleCloudPaymentStatusUpdated(msg));
         break;
-
+    case 'custom_com.mattermost.apps_refresh_bindings': {
```

Comment:
//maybe add a comment that this case handles the apps framework event (`com.mattermost.apps` might be too opaque)